docs/getting_started.md

```diff
@@ -216,6 +219,97 @@ Here are some resources that can be used to get you started working with `AnnDat
+The following commands can be used to access the ADT expression matrices:
+
+```r
+# the raw ADT counts matrix
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# the log normalized ADT counts matrix
+# if this value is NULL, it means normalization failed
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
+
+It is important to be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts.
+The `adt_scpca_filter` column indicates which cells should be removed before proceeding with downstream analyses of the ADT data, as determined by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+Specifically, this process identified cells with high levels of ambient concentration and/or, if available, high levels of negative/isotype control ADTs for removal.
+Cells are labeled either as `"Keep"` (cells to retain) or `"Remove"` (cells to filter out).
+
+To filter cells based on this column, use the following command:
+```r
+# Filter cells based on ADT QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter == "Keep")]
+```
+
+It is also important to be aware that the normalized ADT expression matrix only contains values for cells labeled as `"Keep"` in the `adt_scpca_filter` column.
+Any cells labeled `"Remove"` have `NA` values normalized expression matrix (see
+{ref}`processed adt data <processing_information:Processed ADT data>` for more details).
+
+As dimension reduction requires all known values (i.e., no `NA`s), filtering cells as shown above is critical before proceeding to downstream analyses on ADT counts.
+
+To perform principal components analysis (PCA) on the ADT counts, use the following `scater` command _after_ filtering has been performed:
+
+```r
+# Calculate PCA on the ADT counts
+processed_sce <- scater::runPCA(altExp(processed_sce))
+```
+
+To perform UMAP on this resulting PCA matrix, use the following `scater` command:
+
+```r
+# Calculate UMAP on the ADT counts
+processed_sce <- scater::runUMAP(altExp(processed_sce))
+```
+### Filtering cells based on ADT quality control
+
+In both the `filtered.rds` and `processed.rds` files, cells are flagged for removal based on ADT-level QC statistics in the alternative experiment's `discard` column as calculated by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
+This column contains values `FALSE` for cells that should not be removed and `TRUE` for cells that should be removed.
+
+If starting with the `filtered.rds` file instead, cells will need to be filtered based on ADT counts using the following commands:
+
+```r
+# First, you can see cells should be removed (`TRUE`) vs. kept (`FALSE`)
+altExp(filtered_sce)$discard
```

Comment:
Meant to include this in my earlier review, but let's not encourage people to print a _very_ long vector?

Reply:
I modified this section, for now, to be less prescriptive about `discard` and therefore this code example is also incidentally removed https://github.com/AlexsLemonade/scpca-docs/pull/115/commits/cfc745393f0f3e58d2bdf532c7aebac88f031b32